src/main/java/reactor/netty/tcp/TcpClient.java

```diff
@@ -165,7 +178,9 @@ public final TcpClient remoteAddress(Supplier<? extends SocketAddress> remoteAdd
 	 * enriched bootstrap.
 	 *
 	 * @return a new {@link TcpClient}
+	 * @deprecated  as of 0.9.10. Use the methods exposed on {@link TcpClient} level.
```

Comment:
should probably have a note along the lines of "will be removed in 1.1.0"?

Reply:
This method is kept in master for R2DBC, so I'll add 1.1.0

Reply:
done